components/Common/BlogPostCard/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { useMemo } from 'react';
+import type { ComponentProps, FC } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import AvatarGroup from '@/components/Common/AvatarGroup';
+import Preview from '@/components/Common/Preview';
+
+import styles from './index.module.css';
+
+const dateTimeFormat = new Intl.DateTimeFormat(navigator.language, {
+  day: 'numeric',
+  month: 'short',
+  year: 'numeric',
+});
+
+type Author = {
+  fullName: string;
+  src: ComponentProps<typeof AvatarGroup>['avatars'][number]['src'];
+};
+
+export type BlogPostCardProps = {
```

Comment:
```suggestion
type BlogPostCardProps = {
```
We don't export ComponentProps as they in general can be inferred due `ComponentProps<typeof Component>`

Reply:
Thank you! I actually thought so, and that's why I ended up using `ComponentProps<typeof Component>`, but forgot to remove this `export`.